spec/features/add_unassigned_nodes_feature_spec.rb

```diff
@@ -47,6 +47,9 @@
 
   scenario "An user check all nodes at once to be added", js: true do
     using_wait_time 5 do
+      # wait for all minions to be there
```

Comment:
Correct me if I'm wrong @vitoravelino, but I think the `using_wait_time` is redundant, since the have_content call already guarantees that.